e3sm_diags/derivations/derivations.py

```diff
@@ -693,14 +821,21 @@
     "RELHUM": {
         ("hur",): lambda hur: convert_units(hur, target_units="%"),
         ("RELHUM",): lambda relhum: convert_units(relhum, target_units="%"),
+        ("RelativeHumidity",): lambda relhum: convert_units(
+            relhum, target_units="%"
+        ),  # EAMxx
```

Comment:
Does "target units" imply it will convert the current units to `%`?  Note that in EAMxx RelativeHumidity has units of fraction so is bound from 0 to 1.

Reply:
Thank you for reviewing! Yes, the `convert_units` will convert from fraction to `%`.  I tested the variable at 850mb and can confirm that the derivation works as intended, see example [here](https://portal.nersc.gov/cfs/e3sm/zhang40/tests/eamxx/eamxx_decadal_1996_1111_edv3/lat_lon/ERA5/ERA5_ext-RELHUM-850-ANN-global.png).